shuup/admin/modules/orders/views/refund.py

```diff
@@ -134,12 +142,13 @@ def _get_line_text(self, line):
             text += " (SKU %s)" % (line.sku)
         return text
 
-    def _get_line_number_choices(self):
+    def _get_line_number_choices(self, lines):
+        refundable_lines_count = sum([int(line.max_refundable_quantity) for line in lines])
```

Comment:
Quanitity and amount are two different things. This likely should counting `line.max_refundable_amount`.